README.md

```diff
@@ -1 +1,81 @@
 # PISP Demo Server (Work in Progress)
+
+A server used to communicate with the Mojaloop network to facilitate account linking and transactions for the [PISP demo app](https://github.com/mojaloop/pisp-demo-app-flutter).
+
+# Quick Setup Guide
+
+1. Firebase SDK Admin Key
+
+Follow the instructions outlined [here](https://firebase.google.com/docs/admin/setup) to obtain a JSON file containing your secret key.
+Rename it to be `serviceAccountKey.json` and put it in `{project-directory}/secret/` so that the path to the file is: `{project-directory}/secret/serviceAccountKey`.
+
+Note that this path is just the default path but can be configured (see [Configurations section](#configurations)).
+
+The contents of the JSON file should look like
+```
+{
+  "type": "service_account",
+  "project_id": "firebase-project-name",
+  "private_key_id": "fjj65be89cb3c6b04bacx9637503373fb8.......",
+  "private_key": "{public key in PEM format}",
+  "client_email": "firebase-adminsdk.....iam.gserviceaccount.com",
+  "client_id": "1234567890",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk.....iam.gserviceaccount.com"
+}
+```
+
+2. Environment Variables
+
+Create a `.env` file in your local environment.
+Put these lines into that file:
+```
+EXPERIMENTAL_MODE=on
+EXPERIMENTAL_DELAY=5000
```

Comment:
probably should change this into a smaller number :o, the delay is 5 seconds. Or perhaps you can also remove this since the default is 1 or 2 secs if I remember correctly